Add CONTRIBUTING.md document

carlosgalvezp opened the pull request:
In order to have a common view on the workflow.

carlosgalvezp committed:
Add CONTRIBUTING.md document

In order to have a common view on the workflow.

idanov commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,130 @@
+
+To setup your environment, follow these steps:
+
+1. Go to the [`CarND-Eurobots/CarND-Capstone`](https://github.com/CarND-EuroBots/CarND-Capstone) repository and click on `Fork`. This will create a fork of the repo in
```
Is there an option to open pull requests without forking the project? Could we just create our feature branches in the main repository and open pull requests there?

carlosgalvezp replied:
Sure! The Fork option is what I've seen in many projects but I guess in our case it's fine to just create branches in the main repository. I also find it quite cumbersome to have a Fork especially when updating from upstream.

I'll change that!

carlosgalvezp replied:
Done :)

idanov replied:
Great!

carlosgalvezp committed:
Remove the need of forking the repository

Now it will be possibly to submit changes directly to
the team's repository (always through a Pull Request),
without needing to fork the repository.
